### ISSUE TITLE
Implement a "detached" version of `sss_create_shares`

### ISSUE THREAD

**Issue by dsprenkels** [dsprenkels/sss#1]
For performance reasons you may want to generate a lot of `sss_Share`s, but write the ciphertext that is being generated only once. This is currently not supported by the API. Perhaps we should write a "detached" version of `sss_create_shares` (and `sss_combine_shares`) to make this possible.

**Comment by dsprenkels** [comment]
Performance-wise, this is not really necessary. `memcpy`'ing 64 bytes takes around 8 cycles on modern Intel processors.

Wontfix.
